### PR TITLE
Todo list 2nd branch

### PR DESCRIPTION
**Pull request by husush23**
In this  branch I implemented some functionalities like:
Deleting todo
Added new todo
editing todo
Added to the local storage

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    } else {
+      todos.push({
+        value: todoValue,
+        checked: false,
+      });
+    }
+
+    todoInput.value = '';
+  }
+}
```
Comment (anagudelogu) [inline src/index.js:66]: - [ ] You did a good job working on this milestone! 💯 However, according to the requirements:

image

Please consider moving the new functionality to a new Js file. 🤓 🔧

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        ...todo,
+        value: index === editTodoId ? todoValue : todo.value,
+      }));
+      editTodoId = -1;
+    } else {
+      todos.push({
+        value: todoValue,
+        checked: false,
+      });
+    }
```
Comment (anagudelogu) [inline src/index.js:62]: - [ ] According to the requirements:

image

Please consider renaming your `checked` property as `completed` and adding a new `index` property which is going to be useful to sort your to-dos! 🤓 🔧

**Review by anagudelogu — CHANGES_REQUESTED** [review]
# STATUS:  CHANGES REQUIRED  ♻️ 🚧 

Hi @husush23, 
I'm Andres.

While you have done a great job working on this milestone 💻 , there are some issues that you still need to work on to go to the next project. ♻️ 

You are almost there, keep up the good work! 🚀 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @husush23,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**